lib/features/email/presentation/widgets/calendar_event/calendar_event_action_banner_widget.dart

```diff
@@ -73,15 +72,6 @@ class CalendarEventActionBannerWidget extends StatelessWidget {
                   ]
                 )
               ),
-              if (calendarEvent.getSubTitleEventAction(context).isNotEmpty)
-                Text(
-                  calendarEvent.getSubTitleEventAction(context),
-                  style: const TextStyle(
-                    color: AppColor.colorSubTitleEventActionText,
-                    fontSize: CalendarEventActionBannerStyles.subTileTextSize,
```

Comment:
Please also remove `colorSubTitleEventActionText ` & `subTileTextSize` if not used

Reply:
`AppColor.colorSubTitleEventActionText` is still in used. `CalendarEventActionBannerStyles.subTileTextSize` removed